Regression in multi-rune glyph rendering

Tests børken on master. The render method should only advance the cursor by one cell after drawing a multi-rune glyph.

```
--- FAIL: TestRenderMultiRuneCell (0.00s)
	assertions.go:237: 
                        
	Error Trace:	render_test.go:21
		
	Error:		Not equal: []byte{0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0xd, 0x1b, 0x5b, 0x31, 0x43, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb} (expected)
			        != []byte{0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0x1b, 0x5b, 0x32, 0x43, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb} (actual)
			
			Diff:
			--- Expected
			+++ Actual
			@@ -1,4 +1,4 @@
			-([]uint8) (len=21 cap=32) {
			- 00000000  f0 9f 91 8d f0 9f 8f bb  0d 1b 5b 31 43 f0 9f 91  |..........[1C...|
			- 00000010  8d f0 9f 8f bb                                    |.....|
			+([]uint8) (len=20 cap=32) {
			+ 00000000  f0 9f 91 8d f0 9f 8f bb  1b 5b 32 43 f0 9f 91 8d  |.........[2C....|
			+ 00000010  f0 9f 8f bb                                       |....|
			 }
		
--- FAIL: TestRenderBlankAndMultiRuneCell (0.00s)
	assertions.go:237: 
                        
	Error Trace:	render_test.go:33
		
	Error:		Not equal: []byte{0x20, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0xd, 0x1b, 0x5b, 0x32, 0x43, 0x20} (expected)
			        != []byte{0x20, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0x1b, 0x5b, 0x33, 0x43, 0x20} (actual)
			
			Diff:
			--- Expected
			+++ Actual
			@@ -1,3 +1,3 @@
			-([]uint8) (len=15 cap=16) {
			- 00000000  20 f0 9f 91 8d f0 9f 8f  bb 0d 1b 5b 32 43 20     | ..........[2C |
			+([]uint8) (len=14 cap=16) {
			+ 00000000  20 f0 9f 91 8d f0 9f 8f  bb 1b 5b 33 43 20        | .........[3C |
			 }
		
--- FAIL: TestRenderBlankAndMultiRuneCellOver (0.00s)
	assertions.go:237: 
                        
	Error Trace:	render_test.go:45
		
	Error:		Not equal: []byte{0x20, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0xd, 0x1b, 0x5b, 0x32, 0x43, 0x20} (expected)
			        != []byte{0x20, 0xf0, 0x9f, 0x91, 0x8d, 0xf0, 0x9f, 0x8f, 0xbb, 0x1b, 0x5b, 0x33, 0x43, 0x20} (actual)
			
			Diff:
			--- Expected
			+++ Actual
			@@ -1,3 +1,3 @@
			-([]uint8) (len=15 cap=16) {
			- 00000000  20 f0 9f 91 8d f0 9f 8f  bb 0d 1b 5b 32 43 20     | ..........[2C |
			+([]uint8) (len=14 cap=16) {
			+ 00000000  20 f0 9f 91 8d f0 9f 8f  bb 1b 5b 33 43 20        | .........[3C |
			 }
```

--- a/internal/cops/display/display.go
+++ b/internal/cops/display/display.go
@@ -212,7 +212,11 @@ func RenderOver(buf []byte, cur Cursor, over, under *Display, renderColor ColorM
 			if dy := pt.Y - cur.Position.Y; dy > 0 {
 				buf, cur = cur.linedown(buf, dy)
 			}
-			if dx := pt.X - cur.Position.X; dx > 0 {
+			if cur.Position.X < 0 {
+				buf = append(buf, "\r"...)
+				cur.Position.X = 0
+				buf, cur = cur.right(buf, pt.X)
+			} else if dx := pt.X - cur.Position.X; dx > 0 {
 				buf, cur = cur.right(buf, dx)
 			}
 			buf, cur = renderColor(buf, cur, of, ob)

